src/OpenIddict.Client.WebIntegration/OpenIddictClientWebIntegrationProviders.xml

```diff
@@ -899,6 +899,46 @@
     </Environment>
   </Provider>
 
+  <!--
+                                                    ▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄▄
+                                                    ██ ████ ▄▄▀██ ▄▄▀██ █▀▄██
+                                                    ██ ████ ▀▀ ██ ▀▀▄██ ▄▀███
+                                                    ██ ▀▀ █ ██ ██ ██ ██ ██ ██
+                                                    ▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀
+  -->
+
+  <Provider Name="Lark" DisplayName="Lark (Feishu)" Id="0822ff2a-c4e7-4e1d-a797-bb24632b3f1a"
+            Documentation="https://open.feishu.cn/document/common-capabilities/sso/web-application-sso/web-app-overview?lang=en-US">
```

Comment:
Since global is now the default region, should the docs point to Lark?

Reply:
Yes, my overlook.